sdk/datamigration/Azure.ResourceManager.DataMigration/samples/Generated/Samples/Sample_DatabaseMigrationSqlDBResource.cs

```diff
@@ -230,7 +230,7 @@ public async Task Update_CreateOrUpdateDatabaseMigrationResourceWithMinimumParam
         public async Task Cancel_StopOngoingMigrationForTheDatabase()
         {
             // Generated from example definition: specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2022-03-30-preview/examples/SqlDbCancelDatabaseMigration.json
-            // this example is just showing the usage of "DatabaseMigrationsSqlDb_Cancel" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "DatabaseMigrationsSqlDb_cancel" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `cancel`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2023-07-15-preview/sqlmigration.json#L632